crates/agent/src/api/create_data_plane.rs

```diff
@@ -197,7 +197,7 @@ pub async fn create_data_plane(
         broker_address,
         reactor_address,
         &hmac_keys,
-        serde_json::to_value(encrypted_hmac_keys).unwrap(),
```

Comment:
We actually implement the `sqlx::{Decode,Encode}` traits for `models::RawValue` so that it can be used directly for `JSON` columns. So technically this could have also been fixed by removing this conversion to `Value`. I see zero reason to change what you have here, but just wanted to point out [the `sqlx_json` call](https://github.com/estuary/flow/blob/a53a7825b824e980389d401e2b7b57f6f5405a94/crates/models/src/raw_value.rs#L81) that provides the trait impls, since we use that on a number of different types in `models`.